ui/apps/platform/src/Containers/Administration/Events/AdministrationEventPage.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { ReactElement, useEffect, useState } from 'react';
+import {
+    Alert,
+    Breadcrumb,
+    BreadcrumbItem,
+    Bullseye,
+    Flex,
+    PageSection,
+    Spinner,
+    Title,
+} from '@patternfly/react-core';
+
+import BreadcrumbItemLink from 'Components/BreadcrumbItemLink';
+import PageTitle from 'Components/PageTitle';
+import { AdministrationEvent, getAdministrationEvent } from 'services/AdministrationEventsService';
+import { getAxiosErrorMessage } from 'utils/responseErrorUtils';
+import { administrationEventsBasePath } from 'routePaths';
+
+import { getLevelText } from './AdministrationEvent';
+import AdministrationEventDescription from './AdministrationEventDescription';
+
+export type AdministrationEventPageProps = {
+    id: string;
+};
+
+function AdministrationEventPage({ id }: AdministrationEventPageProps): ReactElement {
+    const [isLoading, setIsLoading] = useState(false);
+    const [event, setEvent] = useState<AdministrationEvent | null>(null);
+    const [errorMessage, setErrorMessage] = useState('');
+
+    useEffect(() => {
```

Comment:
As a potential alternative, there is the `hooks/useRestQuery` hook that abstracts away most of the network call boilerplate.

Reply:
Thank you for reminding me which hook!

I will write first draft up through filtering and fetching to get a realistic comparison of benefits.